__tests__/app.test.js

```diff
@@ -110,3 +110,47 @@ describe('/api/articles', () => {
       });
   });
 });
+
+describe('/api/articles/:article_id/comments', () => {
+  test('GET:200 sends an array of comments for the given article_id to the client', () => {
+    return request(app)
+      .get('/api/articles/9/comments')
+      .expect(200)
+      .then((response) => {
+        expect(response.body.comments.length).toBe(2);
+        response.body.comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe('number');
+          expect(typeof comment.body).toBe('string');
+          expect(typeof comment.votes).toBe('number');
+          expect(typeof comment.author).toBe('string');
+          expect(typeof comment.article_id).toBe('number');
+          expect(typeof comment.created_at).toBe('string');
+        });
```

Comment:
This is really good but we should actually be checking that the article id for every comment is specifically 9 as that is what we are wanting.